src/main/java/org/myrobotlab/inmoov/Utils.java

```diff
@@ -16,13 +16,9 @@
   /**
    * This method will load a python file into the python interpreter. 
    */
-  public static boolean loadPythonFile(String file, String intanceName) {
+  public static boolean loadPythonFile(String file) {

```

Comment:
I'm not sure this is safe in a static context.  Also, it doesn't check if the python service is up and running.. and.. it assumes that the python instance is named "python"  (though, that's a pretty safe assumption.

Reply:
intanceName parameter was not related to python but inmoov instance. To fix a possible issue if user set inmoov service name other than "i01" . deleted will return to it later…
checked also for null py instance